lexxpluss_apps/src/led_controller.hpp

```diff
@@ -36,7 +36,7 @@ struct msg {
     msg(uint32_t pattern, uint32_t interrupt_ms) :
         pattern(pattern), interrupt_ms(interrupt_ms) {}
     msg(const char *str) {
-        if      (strcmp(str, "emergency_stop")  == 0) pattern = EMERGENCY_STOP;
+        if      (strcmp(str, "safety_pause")  == 0) pattern = EMERGENCY_STOP;
```

Comment:
And, fix the parts where EMERGENCY_STOP is being used.
```suggestion
        if      (strcmp(str, "safety_pause")  == 0) pattern = SAFETY_PAUSE;
```